layers/+lang/javascript/README.org

```diff
@@ -149,3 +159,14 @@ You can check more [[https://github.com/mooz/js-doc/][here]]
 | Key Binding | Description                        |
 |-------------+------------------------------------|
 | ~SPC m h p~ | Get the path of the value at point |
+
+** REPL (skewer-mode)
+
+| Key Binding | Description                                                             |
+|-------------+-------------------------------------------------------------------------|
+| ~SPC m s i~ | prepares the skewer environment - run this before running REPL commands |
+| ~SPC m s r~ | starts/goes to the skewer REPL                                          |
```

Comment:
This key bindings is reserved to send the current region to the REPL.
I think you want `SPC m s s` here.
see https://github.com/syl20bnr/spacemacs/blob/master/doc/CONVENTIONS.org#repls